synfig-studio/src/gui/canvasview.cpp

```diff
@@ -733,6 +733,7 @@ void CanvasView::deactivate()
 
 void CanvasView::present()
 {
+	this->show(); // gtk_widget_show is also checks visibility, so there is no need to call `is_visible()`
```

Comment:
Why the `this->`? ;)

Anyway there's a typo in comments: "gtk_widget_show `is` also checks visibility" -> "gtk_widget_show also checks visibility"

Reply:
Fixed.